spec/models/stamp_duty_spec.rb

```diff
@@ -354,5 +354,37 @@
         its(:total_due) { is_expected.to eq(2_328_750) }
       end
     end
+
+    context 'when house price is not easily divisible' do
```

Comment:
`context 'when house price is not easily divisible' do` --> `context 'when house price has result with decimal values' do`

Since _easily divisible_ can assume many definitions.